st2common/st2common/models/db/liveaction.py

```diff
@@ -69,13 +70,23 @@ class LiveActionDB(stormbase.StormFoundationDB):
         help_text='Callback information for the on completion of action execution.')
     runner_info = me.DictField(
         default={},
-        help_text='Reference to the runner that executed this liveaction.')
+        help_text='Information about the runner which executed this live action (hostname, pid).')
     notify = me.EmbeddedDocumentField(NotificationSchema)
 
     meta = {
         'indexes': ['-start_timestamp', 'action']
     }
 
+    def to_serializable_dict(self, mask_secrets=False):
+        result = super(LiveActionDB, self).to_serializable_dict(mask_secrets=mask_secrets)
+
+        if mask_secrets:
+            # TODO: This sucks, but it's only non slow approach
+            del result['parameters']
```

Comment:
I am confused about this. Won't this delete the `parameters` section completely?